### PR TITLE
[2.13.x] DDF-4422 Making LDAP logging more verbose

### PR DESCRIPTION
**Pull request by Kjames5269**
#### What does this PR do?
Improves logging when there is an LDAP authentication issue.

#### Who is reviewing it? 

@blen-desta @bakejeyner  

#### Ask 2 committers to review/merge the PR and tag them here.

@tbatie @brjeter 
#### How should this be tested?
1. Configure LDAP for Auth and a data store as shown below:
ldap-config

2. Login with the ldap credentials
3. The data store should be incorrectly setup so you'll see an error that the user failed to login, and the attributes were not found.

#### What are the relevant tickets?
[DDF-4422](https://codice.atlassian.net/browse/DDF-4422)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java [hunk 1]
```diff
@@ -409,9 +416,7 @@ private void installLdapConnectionPool(String connectionPoolId) {
   public boolean login() throws LoginException {
     boolean isLoggedIn;
     String message =
```
Comment (tbatie) [inline platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java]: ✏️ Looks like this could be static final

**Comment by clockard** [comment]
build now

**Comment by Kjames5269** [comment]
build now

**Comment by bakejeyner** [comment]
Hero ✔️

**Comment by Kjames5269** [comment]
build now
(aborted due to changes)

**Pull request merged**